src/lib/components/module/FunctionCard.tsx

```diff
@@ -4,22 +4,22 @@ import { Flex, Text } from "@chakra-ui/react";
 import { DotSeparator } from "../DotSeparator";
 import { CustomIcon } from "../icon";
 import { Tooltip } from "lib/components/Tooltip";
+import type { ExposedFunction, Visibility } from "lib/types";
 
 interface FunctionCardProps {
-  isView?: boolean;
-  disabled?: boolean;
-  visibility?: "public" | "friends" | "script";
+  exposedFn: ExposedFunction;
+  onFunctionSelect: (fn: ExposedFunction) => void;
 }
 
-const getIcon = (visibility: FunctionCardProps["visibility"]) => {
+const getIcon = (visibility: Visibility) => {
   switch (visibility) {
-    case "friends":
-      return <CustomIcon name="friends" color="text.dark" boxSize={3} />;
+    case "friend":
+      return <CustomIcon name="friends" color="gray.600" boxSize={3} />;
     case "script":
-      return <CustomIcon name="code" color="text.dark" boxSize={3} />;
+      return <CustomIcon name="code" color="gray.600" boxSize={3} />;
     case "public":
     default:
-      return <CustomIcon name="website" color="text.main" boxSize={3} />;
+      return <CustomIcon name="website" color="gray.600" boxSize={3} />;
   }
 };
```

Comment:
change to function that return the name instead of a component?

Reply:
done in #521